azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -459,3 +461,15 @@ def new_guid(self) -> UUID:
         self._new_uuid_counter += 1
         guid = uuid5(NAMESPACE_URL, guid_name)
         return guid
+
+    def _pretty_print_history(self) -> str:
```

Comment:
I'm a bit confused. This looks like it returns a `str`, but the return type appears to be a list of `str`?

Reply:
whoops, you're right. Just fixed it in my last commit to merely return a string.

I skipped the typechecking step here because I was already tackling it in my other PR, sorry about that.